lib/Sema/DerivedConformanceParameterized.cpp

```diff
@@ -374,12 +382,35 @@ static Type deriveParameterized_Parameters(DerivedConformance &derived) {
   }
   parametersDecl->setValidationToChecked();
 
-  // Add conformance to the ParameterGroup protocol, if possible.
-  // The ParameterGroup protocol requirements will be derived.
-  if (DerivedConformance::canDeriveParameterGroup(parametersDecl)) {
-    TypeLoc inherited[1] = {paramGroupType};
-    parametersDecl->setInherited(C.AllocateCopy(inherited));
+  // Declare conformance to `VectorNumeric` (if possible), `Differentiable`
+  // and `KeyPathIterable` protocols. Conformances will be derived.
+  SmallVector<TypeLoc, 3> inherited {differentiableType, keyPathIterableType};
```

Comment:
This is correct and crucial!
Inheriting `keyPathIterableType` is important to leverage new `KeyPathIterable` protocol.
Inheriting `differentiableType ` is important but it blocked by `Differentiable` derived conformances. I'm working on it and will push ASAP.